### ISSUE TITLE
What's the correct way to deduplicate sanitized addresses?

### ISSUE THREAD

**Issue by xxx** [pboling/sanitize_email#64]
Hi, thank you for creating this gem.

I'm curious about what the best way to deduplicate sanitized addresses is. Right now we're sanitizing in our staging environment to prevent external sends, but replaces of multiple addresses end up with the same address multiple times in the final result. This causes DuplicatePersonalization errors from Sendgrid, which prohibits this. Right now I'm doing this with the following monkey patch:

```ruby
module SanitizeEmail
  class Bleach
    class << self
      alias original_delivering_mail delivering_email
    end

    def self.delivering_email(message)
      original_delivering_mail(message)

      message.to = message.to&.uniq
      message.cc = message.cc&.uniq
      message.bcc = message.bcc&.uniq
    end
  end
end
```

Is there a more correct way to do this?

**Comment by pboling** [comment]
I'll add a built-in de-dup for the next release.  Thanks for the bug report and idea!

**Comment by pboling** [comment]
Fixed in 01c80bb
